packages/FlexGrid/Col/Col.jsx

```diff
@@ -5,52 +5,111 @@ import { Subscriber } from 'react-broadcast'
 import { Col as ReactFlexboxGridCol } from 'react-flexbox-grid'
 
 import safeRest from '../../../shared/utils/safeRest'
+import { deprecate } from '../../../shared/utils/warn'
 import styles from './Col.modules.scss'
 
-const removeProps = ({
-  xs,
-  sm,
-  md,
-  lg,
-  xl,
-  xsOffset,
-  smOffset,
-  mdOffset,
-  lgOffset,
-  xlOffset,
-  ...rest
-}) => safeRest(rest)
-
-const Col = ({ span, offset, children, ...rest }) => (
-  <Subscriber channel="flex-grid">
-    {gutter => (
-      <ReactFlexboxGridCol
-        {...removeProps(rest)}
-        xs={span || true}
-        xsOffset={offset}
-        className={gutter ? styles.padding : styles.gutterless}
-      >
-        {children}
-      </ReactFlexboxGridCol>
-    )}
-  </Subscriber>
-)
+const Col = ({ children, span, offset, ...rest }) => {
+  if (offset) {
+    deprecate(
+      'core-flex-grid',
+      `The offset prop is deprecated due to the addition of the new responsive
+    offset props.
+    Replace offset in your Col definition with xsOffset for identical functionality.`
+    )
+  }
+  if (span) {
+    deprecate(
+      'core-flex-grid',
+      `The span prop is deprecated due to the addition of the new responsive props.
+      Replace span in your Col definition with xs for identical functionality.`
+    )
+  }
 
+  const props = rest
+
+  if (offset && !props.xsOffset) props.xsOffset = offset
```

Comment:
After creating the `props` variable and adding `xsOffset` to it here, you don't do anything with it....  is the intention to pass `props` into `safeRest` now?

Reply:
Yes, good catch

Reply:
Good catch, I'll make that change. Turns out I didn't notice it due to the behaviour of object copying:

![image](https://user-images.githubusercontent.com/12798751/38422544-20dcb722-3979-11e8-9088-c36d57f123a0.png)

Reply:
haha yep! I did the same exact thing to check it yesterday too!

If you want to copy an object without effecting the original, the best way is:

```js
const copyOfObj = Object.assign({}, obj)

// or using the spread syntax
const copyOfObj = {...obj}
```